.github/templates/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+*Descriptive Summary of your Bugfix, Feature, or Refactoring.*
+
+## List of Added or Changed Components
+- feature a added to module bar
+- bug b fixed in foo.rs
```

Comment:
I think we should keep the template fairly minimal, so I think this "List of Added or Changed Components" section can be dropped. The "Descriptive Summary" is the most interesting part, and any details can be covered in commit messages and the change log.

Reply:
Alright!